src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_ISwitchStatement.vb

```diff
@@ -0,0 +1,876 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports Microsoft.CodeAnalysis.Test.Utilities
+Imports Roslyn.Test.Utilities
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests.Semantics
+
+    Partial Public Class IOperationTests
+        Inherits SemanticModelTestBase
+
+        <CompilerTrait(CompilerFeature.IOperation)>
+        <Fact, WorkItem(22006, "https://github.com/dotnet/roslyn/issues/22006")>
+        Public Sub ISwitchStatement_Simple()
+            Dim source = <![CDATA[
+Class Test
+    Sub M(number As Integer)
+        Select Case number'BIND:"Select Case number"
+            Case 0
+                Exit Select
+        End Select
+    End Sub
+End Class]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+ISwitchStatement (1 cases) (OperationKind.SwitchStatement) (Syntax: 'Select Case ... End Select')
+  Switch expression: 
+    IParameterReferenceExpression: number (OperationKind.ParameterReferenceExpression, Type: System.Int32) (Syntax: 'number')
+  Sections:
+      ISwitchCase (1 case clauses, 1 statements) (OperationKind.SwitchCase) (Syntax: 'Case 0 ... Exit Select')
+          Clauses:
+              ISingleValueCaseClause (CaseKind.SingleValue) (OperationKind.CaseClause) (Syntax: '0')
+                Value: 
+                  ILiteralExpression (OperationKind.LiteralExpression, Type: System.Int32, Constant: 0) (Syntax: '0')
+          Body:
+              IBlockStatement (1 statements) (OperationKind.BlockStatement) (Syntax: 'Case 0 ... Exit Select')
+                IBranchStatement (BranchKind.Break, Label: exit) (OperationKind.BranchStatement) (Syntax: 'Exit Select')
```

Comment:
>Label: exit [](start = 52, length = 11)

Do we have a test that exposes the destination for this branch (the corresponding ILabeledStatement I guess)?